<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileBrowserWindowWindow</class>
 <widget class="QDockWidget" name="FileBrowserWindowWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Browser</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="layout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <action name="actionHome">
   <property name="icon">
    <iconset resource="../../../../../res/ui.qrc">
     <normaloff>:/oxygen/places/user-home.png</normaloff>:/oxygen/places/user-home.png</iconset>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
   <property name="statusTip">
    <string>Go to the home folder</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="icon">
    <iconset resource="../../../../../res/ui.qrc">
     <normaloff>:/oxygen/actions/go-previous.png</normaloff>:/oxygen/actions/go-previous.png</iconset>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
   <property name="statusTip">
    <string>Go to the previous folder or file</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="../../../../../res/ui.qrc">
     <normaloff>:/oxygen/actions/go-next.png</normaloff>:/oxygen/actions/go-next.png</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
   <property name="statusTip">
    <string>Go to the next folder or file</string>
   </property>
  </action>
  <action name="actionParent">
   <property name="icon">
    <iconset resource="../../../../../res/ui.qrc">
     <normaloff>:/oxygen/actions/go-up.png</normaloff>:/oxygen/actions/go-up.png</iconset>
   </property>
   <property name="text">
    <string>Parent</string>
   </property>
   <property name="statusTip">
    <string>Go to the parent folder</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../../../res/ui.qrc"/>
 </resources>
 <connections/>
</ui>
